scripts/Muon/GUI/MuonAnalysis/plot_widget/plot_time_fit_pane_model.py

```diff
@@ -0,0 +1,61 @@
+# Mantid Repository : https://github.com/mantidproject/mantid
+#
+# Copyright &copy; 2021 ISIS Rutherford Appleton Laboratory UKRI,
+#   NScD Oak Ridge National Laboratory, European Spallation Source,
+#   Institut Laue - Langevin & CSNS, Institute of High Energy Physics, CAS
+from Muon.GUI.Common.plot_widget.data_pane.plot_group_pair_model import PlotGroupPairModel
+from Muon.GUI.Common.ADSHandler.workspace_naming import *
+
+
+class PlotTimeFitPaneModel(PlotGroupPairModel):
+
+    def __init__(self, context):
+        super().__init__(context,"Fit Data")
+        self.context.plot_panes_context[self.name].set_defaults([0.,15.], [-0.3, 0.3])
+
+    @staticmethod
+    def get_fit_workspace_and_indices(fit, with_diff=True):
```

Comment:
Are these functions the same as the ones in the frequency pane model or is there a slight difference? Is there any common code we can separate and then reuse?

Reply:
the code is similar. However, I didn't like the frequency code having the if for tf asymmetry mode.